index.js

```diff
@@ -1 +1,87 @@
-// implement your API here
+const express = require('express');
+const db = require('./data/db.js');
+
+const server = express();
```

Comment:
this is just a small and unnecessary, but cool optimization trick: since `express` being invoked here is just a reference to what you captured on line 1, you can actually merge the two and cut out the middleman like `const server=require('express')()`. of course, so long as you need to use express.json as middleware down below it doesn't really save you time writing that again, but something cool nonetheless